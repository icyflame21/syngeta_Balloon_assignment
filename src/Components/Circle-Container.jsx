import React from 'react'

export const Circle_Container = () => {
  return (
    <div className="circles-container">
        <p className="text-center">5 Circles</p>
        <div className="circle" id="circle-1">1</div>
        <div className="circle" id="circle-2">2</div>
        <div className="circle" id="circle-3">3</div>
        <div className="circle" id="circle-4">4</div>
        <div className="circle" id="circle-5">5</div>
      </div>
  )
}
