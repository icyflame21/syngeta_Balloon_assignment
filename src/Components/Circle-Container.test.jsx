import { shallow } from 'enzyme';
import React from 'react'
import { Circle_Container } from "./Circle-Container";


it('check is Circle_Container is loaded', () => {
  const component = shallow(<Circle_Container />);
  expect(component).toBeTruthy()
  expect(component.getElements()).toMatchSnapshot();
});

it('test to match snapshot', () => {
    const component = shallow(<Circle_Container />);
    expect(component.getElements()).toMatchSnapshot();
  });
