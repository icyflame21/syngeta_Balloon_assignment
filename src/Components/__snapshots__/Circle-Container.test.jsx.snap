// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check is Empty_Container is loaded 1`] = `
Array [
  <div
    className="circles-container"
>
    <p
        className="text-center"
    >
        5 Circles
    </p>
    <div
        className="circle"
        id="circle-1"
    >
        1
    </div>
    <div
        className="circle"
        id="circle-2"
    >
        2
    </div>
    <div
        className="circle"
        id="circle-3"
    >
        3
    </div>
    <div
        className="circle"
        id="circle-4"
    >
        4
    </div>
    <div
        className="circle"
        id="circle-5"
    >
        5
    </div>
</div>,
]
`;

exports[`test to match snapshot 1`] = `
Array [
  <div
    className="circles-container"
>
    <p
        className="text-center"
    >
        5 Circles
    </p>
    <div
        className="circle"
        id="circle-1"
    >
        1
    </div>
    <div
        className="circle"
        id="circle-2"
    >
        2
    </div>
    <div
        className="circle"
        id="circle-3"
    >
        3
    </div>
    <div
        className="circle"
        id="circle-4"
    >
        4
    </div>
    <div
        className="circle"
        id="circle-5"
    >
        5
    </div>
</div>,
]
`;
